contracts/DAOTreasury.cdc

```diff
@@ -8,22 +8,38 @@ pub contract DAOTreasuryV3 {
   pub let TreasuryStoragePath: StoragePath
   pub let TreasuryPublicPath: PublicPath
 
-  // Events
+  // ------- Events ------
+
+  // Treasury
   pub event TreasuryInitialized(initialSigners: [Address], initialThreshold: UInt)
-  pub event ProposeAction(actionUUID: UInt64, proposer: Address)
-  pub event ExecuteAction(actionUUID: UInt64, proposer: Address)
-  pub event DepositVault(vaultID: String)
-  pub event DepositCollection(collectionID: String)
-  pub event DepositTokens(identifier: String)
-  pub event DepositNFT(collectionID: String, nftID: UInt64)
-  pub event WithdrawTokens(vaultID: String, amount: UFix64)
-  pub event WithdrawNFT(collectionID: String, nftID: UInt64)
-  pub event DestroyVault(vaultID: String)
-  pub event DestroyCollection(collectionID: String)
-
-
-  // Interfaces + Resources
+
+  // Actions
+  pub event ActionProposed(treasuryUUID: UInt64, actionUUID: UInt64, proposer: Address, actionView: MyMultiSigV3.ActionView)
+  pub event ActionExecuted(treasuryUUID: UInt64, actionUUID: UInt64, executor: Address, actionView: MyMultiSigV3.ActionView, signerResponses: {Address: UInt})
+  pub event ActionDestroyed(treasuryUUID: UInt64, actionUUID: UInt64, signerResponses: {Address: UInt})
+  pub event ActionApprovedBySigner(treasuryUUID: UInt64, address: Address, uuid: UInt64, signerResponses: {Address: UInt})
+  pub event ActionRejectedBySigner(treasuryUUID: UInt64, address: Address, uuid: UInt64, signerResponses: {Address: UInt})
```

Comment:
for above two events, since they already have `treasuryUUID`, can we get rid of `uuid`?

Reply:
`uuid` is the action UUID! we could roll it into `actionView` although I'm not sure if we can filter events in Graffle using params nested in structs... so probably worth keeping it on the action directly.

Reply:
it looks like they both assigned with `self.uuid` and i thought that's the treasury uuid, did i miss anything here?
https://github.com/DapperCollectives/VESSEL/blob/eb56c078bd14ba4b7d1a8e2be2b0cef84fad37bb/contracts/DAOTreasury.cdc#L72

Reply:
oh nope you are totally right, sorry about that. i will change `uuid: self.uuid` -> `actionUUID: action.uuid`